### PR TITLE
nRF7002-EB shield board support

### PR DESCRIPTION
**Pull request by bjarki-andreasen**
Add board support for the nrf7002-EB shield.

### CODE REVIEW

**Review by jukkar — DISMISSED** [review]
LGTM

**Comment by bjarki-andreasen** [comment]
name of image was wrong :)

**Review by jukkar — DISMISSED** [review]

**Review by krish2718 — DISMISSED** [review]

**Comment by bjarki-andreasen** [comment]
Updated to not select  the redundant `NRF_WIFI_COMBINED_BUCKEN_IOVDD_GPIO` which is removed with PR https://github.com/zephyrproject-rtos/zephyr/pull/78485

**Review by krish2718 — DISMISSED** [review]

**Review by jukkar — DISMISSED** [review]

**Review by jfischer-no — DISMISSED** [review]

File: boards/shields/nrf7002eb/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+The shield can be used in any application by setting ``--shield nrf7002eb`` when invoking ``west build``.
+
+Shield Variants
+###############
+
+The nRF7002 EK has a variant which includes the COEX pins. These pins are not be routed to the
+edge-connector on some boards, like earlier revisions of the Thingy53 than v1.0.0.
+
+- ``nrf7002ek``: The default variant.
+- ``nrf7002ek_coex``: Variant which includes the COEX pins.
```
Comment (kartben) [inline boards/shields/nrf7002eb/doc/index.rst:53]: Is it eb or ek? :)
Looks like you'll want to be consistent and also make sure you align the name of the main directory with whatever you end up chosing.

**Review by kartben — CHANGES_REQUESTED** [review]

File: boards/shields/nrf7002eb/doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+SR Co-existence
+###############
+
+The nRF7002 EK supports SR co-existence provided the host board supports it. The SR co-existence
```
Comment (kartben) [inline boards/shields/nrf7002eb/doc/index.rst:58]: ...EB...

There might be more so I trust you will do a full search and replace :)

File: boards/shields/nrf7002eb/doc/index.rst (hunk 1 above)
Comment (bjarki-andreasen) [reply boards/shields/nrf7002eb/doc/index.rst:53]: Nice catch :) its definately supposed to be EB

File: boards/shields/nrf7002eb/doc/index.rst (hunk 2 above)
Comment (bjarki-andreasen) [reply boards/shields/nrf7002eb/doc/index.rst]: You did catch them all :)

**Review by kartben — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Pull request merged**